### ISSUE TITLE
Lesson 16 practice 2: robot scale and position test allow for wrong code

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/learn-gdscript#286]
It seems the compared scale and position value in the tests are incorrect. Code that adds or subtracts to the position and scale can make the practice pass, even if the output isn't the expected one.

![Screenshot from 2022-02-08 12-27-05](https://user-images.githubusercontent.com/12694995/152978305-14231228-bb23-4b40-ba35-cd0972061086.png)

Ping @johnnygossdev

**Comment by johnnygossdev** [comment]
Hm didn't expect that! I'll have a look.

**Comment by johnnygossdev** [comment]
The test was grabbing the wrong node. I neglected to change the test when I refactored to use the new robot scene.

```
func _prepare() -> void:
	robot = _scene_root_viewport.get_child(0)
```

Here's the scene.

![image](https://user-images.githubusercontent.com/3285214/152985455-07c65662-22c2-4b3f-a2b1-9a1b1b155f80.png)

But curiously, the order of children gets mixed up in `Remote` when the tests begin. I assume this is something to do with adding and removing the scene.

![image](https://user-images.githubusercontent.com/3285214/152985660-4abbf561-88bf-446a-bf52-f50bc8321c5a.png)
